src/list.coffee

```diff
@@ -56,6 +57,11 @@ class List extends Command
       tree packages, (pack) =>
         packageLine = pack.name
         packageLine += "@#{pack.version}" if pack.version?
+        if pack.apmInstallSource?.type is 'git'
+          repo = getRepository(pack)
+          shaLine = "##{pack.apmInstallSource.sha.substr(0, 8)}"
```

Comment:
I agree on this in `settings-view` but I'm not so sure about this one; this is to enable this output in `apm list`:

```
Git Packages (1) /Users/kuychaco/.atom/packages
└── atom-dev-tools@0.0.1 (BinaryMuse/atom-dev-tools#19c33bd1)
```

(e.g. the "#" separates the repo and the sha, which is a fairly standard Git thing iirc)